assets/extensions/examples/online-extensions.js

```diff
@@ -58,27 +57,27 @@ angular
 
           system_status_elem_mobile.append(system_status_elem.clone());
           system_status_elem_mobile.css('display', '');
-        }
-      });
-
-
-      extensionRegistry
-        .add('nav-system-status', function() {
-          return [{
-            type: 'dom',
-            node: system_status_elem
-          }];
-        });
 
+          // only add the extension if there is something to show so we
+          // do not generate empty nodes if no issues
+          extensionRegistry
+            .add('nav-system-status', function() {
```

Comment:
if so then yes please clean up all the hackiness around display css for the elements, that was all because of how the old extension service worked to always guarantee the status would show up no matter the order things ran